### PR TITLE
Add postfix-exporter in docker-compose.yml for development.

### PR DESCRIPTION
**Pull request by Gary-H9**
- added volumes to share logs and socker between docker images
- added service smtp_relay_monitoring which uses postfix-exporter
- added blank postfix file as /var/log/mail/postfix must exist prior to smtp_relay_monitoring

### CODE REVIEW

File: smtp-relay/Dockerfile [hunk 1]
```diff
@@ -6,12 +6,13 @@ RUN apk add --no-cache --update postfix bash && \
     # Clean up
     (rm "/tmp/"* 2>/dev/null || true) && (rm -rf /var/cache/apk/* 2>/dev/null || true)
 
-VOLUME [ "/var/spool/postfix", "/etc/postfix" ]
+VOLUME [ "/var/spool/postfix", "/etc/postfix", "/var/log/mail", "/var/spool/postfix" ]
```
Comment (richrace) [inline smtp-relay/Dockerfile:9]: We accidentally added a duplicate `"/var/spool/postfix"`

File: smtp-relay/Dockerfile [hunk 2]
```diff
@@ -6,12 +6,12 @@ RUN apk add --no-cache --update postfix cyrus-sasl ca-certificates bash && \
...
 
 COPY entrypoint.sh /usr/local/bin/
 
 EXPOSE 587 25
 
 ENTRYPOINT ["entrypoint.sh"]
 
-CMD ["postfix", "start-fg"]
\ No newline at end of file
+CMD ["postfix", "start-fg" ]
```
Comment (richrace) [inline smtp-relay/Dockerfile:17]:
```suggestion
CMD ["postfix", "start-fg"]

```

File: smtp-relay-monitoring/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+set -e
+
+FILE=/var/log/mail/postfix
+
+if [ ! -f "$FILE" ]; then
+  mkdir -p "/var/log/mail"
+  touch "$FILE"
+fi
+
+/bin/postfix_exporter --postfix.logfile_path=$FILE
```
Comment (richrace) [inline smtp-relay-monitoring/entrypoint.sh:11]:
```suggestion
/bin/postfix_exporter --postfix.logfile_path=$FILE

```

File: smtp-relay-monitoring/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+FROM jacobbaungard/postfix_exporter:latest
+
+VOLUME [ "/var/log/mail", "/var/spool/postfix" ]
+
+COPY entrypoint.sh /usr/local/bin/
+
+EXPOSE 9154
+
+ENTRYPOINT [ "entrypoint.sh" ]
```
Comment (richrace) [inline smtp-relay-monitoring/Dockerfile:9]:
```suggestion
ENTRYPOINT [ "entrypoint.sh" ]

```

**Review by richrace — COMMENTED** [review]
Also noticed there's not `.PHONY`s in the Makefile do we need to add these?

**Review by mTouhid — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**